go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/containerd/fuse-overlayfs-snapshotter v1.0.2
 	github.com/containerd/go-cni v1.1.9
 	github.com/containerd/go-runc v1.1.0
-	github.com/containerd/nydus-snapshotter v0.8.2
+	github.com/containerd/nydus-snapshotter v0.13.0
```

Comment:
Not sure if we also need to bump nydusd (as in https://github.com/moby/buildkit/pull/3814).

Reply:
Thanks for the PR, it's okay to keep the current nydusd version in buildkit.